pkgs/macro/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: macro
```

Comment:
We have package:macros, this will be a replacement for that right? We should probably try and just re-use that name.